java/ci/Dockerfile.centos7

```diff
@@ -0,0 +1,22 @@
+###
```

Comment:
Missing the copyright and license comment block

Reply:
Added the copyright.